deployment/test_examples.py

```
@@ -11,53 +11,53 @@ def test_model():
         "I feel calm and peaceful right now.",
         "I'm content with how things are going.",
         "I'm hopeful that things will get better.",
         "I'm tired and need some rest."
     ]
-    
+
     print("\n📊 Testing Results:")
     print("=" * 50)
-    
+
     correct_predictions = 0
```

Ruff (0.12.2)

Local variable `correct_predictions` is assigned to but never used

Remove assignment to unused variable `correct_predictions`

(F841)